src/robusta/core/model/runner_config.py

```diff
@@ -81,6 +82,20 @@ class RunnerConfig(BaseModel):
     active_playbooks: Optional[List[PlaybookDefinition]] = []
     alert_relabel: Optional[List[AlertRelabel]] = []
 
+    @root_validator()
+    def ensure_unique_sink_name(cls, val: Dict) -> Dict:
+        print(f"validating {val}")
+        if val.get('sinks_config'):
+            value_set = set()
+            sinksConfig = cast(List[SinkConfigBase], val.get('sinks_config'))
+            for sink_config in sinksConfig:
+                sink_name = sink_config.get_name()
+                if sink_name in value_set:
+                    raise ValueError(f"Sink name \"{sink_name}\" is already defined. Sink names must be unique.")
+                else:
```

Comment:
You don't need `else` here because after raising a ValueError the loop stops

Reply:
Thank you